packages/@react-spectrum/toast/docs/Toast.mdx

```diff
@@ -165,6 +165,14 @@ function Example() {
 }
 ```
 
+## Placement
+
+By default, toasts are displayed at the bottom center of the screen. This can be changed by setting the `placement` prop on the `ToastContainer` to `'top start'`, `'top'`, `'top end'`, `'bottom start'`, `'bottom'`, or `'bottom end'`.
```

Comment:
How does the team feel about a blurb for mfe's that may have multiple ToastContainers?

Reply:
Good catch! I checked and we currently don't talk about multiple ToastContainers in the docs. Do we need a general section on that or continue not documenting it?